pkg/adapter/containers.go

```diff
@@ -607,3 +607,90 @@ func (r *LocalRuntime) Start(ctx context.Context, c *cliconfig.StartValues, sigP
 	}
 	return exitCode, lastError
 }
+
+// PauseContainers removes container(s) based on CLI inputs.
+func (r *LocalRuntime) PauseContainers(ctx context.Context, cli *cliconfig.PauseValues) ([]string, map[string]error, error) {
+	var (
+		ok       = []string{}
+		failures = map[string]error{}
+		ctrs     []*libpod.Container
+		err      error
+	)
+
+	maxWorkers := shared.DefaultPoolSize("pause")
+	if cli.GlobalIsSet("max-workers") {
+		maxWorkers = cli.GlobalFlags.MaxWorks
+	}
+	logrus.Debugf("Setting maximum rm workers to %d", maxWorkers)
+
+	if cli.All {
+		ctrs, err = r.GetRunningContainers()
+	} else {
+		ctrs, err = shortcuts.GetContainersByContext(false, false, cli.InputArgs, r.Runtime)
```

Comment:
Isn't all a bool in this function? Don't need the if check, I think

Reply:
Actually, nevermind - I think we're missing some logic on the `cli.All` if - we should be grabbing only running containers

Reply:
The man page for pause says all should pause all running containers.  So the way I read this is 626 says if all, get all running containers .... seems correct to me?

Reply:
Ah, I see